### ISSUE TITLE
Review scrollbar-stable

### ISSUE THREAD

**Issue by darrenburns** [Textualize/rich#2908]
I have a suspicion that scrollbar-stable doesn't work, or (fairly likely) I don't understand what it does.

**Comment by darrenburns** [comment]
I think it does work, or at least it did work as of a couple of months ago. I believe I use it in my file manager.

**Comment by willmcgugan** [comment]
Yeah, I think I understand it a bit more now. Does seem to do something useful.
